azurerm/resource_arm_public_ip.go

```diff
@@ -60,6 +60,17 @@ func resourceArmPublicIp() *schema.Resource {
 				}, true),
 			},
 
+			"ip_version": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  string(network.IPv4),
+				ForceNew: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					string(network.IPv4),
+					string(network.IPv6),
+				}, false),
```

Comment:
@Krueladin I hope you don't mind - since this PR otherwise LGTM I've pushed a commit to make this case-sensitive